app/Assets/Helpers.php

```diff
@@ -257,4 +257,16 @@ public function data_index_set(int $idx = 0): void
 	{
 		$this->numTab = $idx;
 	}
+
+	/**
+	 * Check if the `exec` function is available.
+	 *
+	 * @return bool
+	 */
+	public function isExecAvailable(): bool
+	{
+		$disabledFunctions = explode(',', ini_get('disable_functions'));
+
+		return function_exists('exec') && !in_array('exec', $disabledFunctions, true);
+	}
```

Comment:
Why do we need this function? Why can we not use the Laravel helper everywhere?

Reply:
What do you mean ?

Reply:
Sorry, my fault.